### ISSUE TITLE
--vmware-boot2docker-url and boot2docker releases

### ISSUE THREAD

**Issue by danielfly** [docker/machine#2096]
When trying to specifying --vmware-boot2docker-url with a simple release url from boot2docker 

```
docker-machine create --driver=vmwarefusion --vmware-boot2docker-url https://github.com/boot2docker/boot2docker/releases/download/v1.8.2/boot2docker.iso test
``` 
Error
```
Error creating machine: Get https://github.com/boot2docker/boot2docker/releases/download/v1.8.2/boot2docker.iso: dial tcp: i/o timeout
```
I almost sure the error is because the github url redirects to a S3 and the application dont follow the redirect.

Please correct me if im using a wrong url from github. :)

Specs
```
Mac 10.10.5
docker-machine version 0.4.1 (e2c88d6)
```

**Comment by dgageot** [comment]
Hi @danielfly, I've just tried this command with master and it works.

```
docker-machine create --driver=vmwarefusion --vmwarefusion-boot2docker-url https://github.com/boot2docker/boot2docker/releases/download/v1.8.2/boot2docker.iso test
```

You might want to try.

**Comment by sazo** [comment]
Hi @dgageot, danielfly here got 2 github accounts. 

Maybe it is fixed in the new version of docker-machine 0.5.0? Do you got a list of know issues for 4.1? :) i got docker-machine 4.1 from the Docker Toolbox.

**Comment by dgageot** [comment]
Closing this issue. This is fixed in 0.5.0

Thanks you for the feedback @sazo
